dev/packages/alpha/cisco/0.0.2/dataset/ios/agent/stream/stream.yml.hbs

```diff
@@ -0,0 +1,13 @@
+input: log
+paths:
+{{#each paths}}
+  - {{this}}
+{{/each}}
+exclude_files: [".gz$"]
+tags: {{this.tags}}
+processors:
+  - add_locale: ~
+  - script:
+      lang: javascript
+      id: cisco_ios
+      file: ${path.home}/module/cisco/ios/config/pipeline.js
```

Comment:
@ph @michalpristas Can you please comment on if this path will be correct? What `path.home` will it take inside the config, the one of the local Filebeat or the agent?